src/Chinchilla/Subscription.cs

```diff
@@ -65,6 +65,12 @@ public IWorkersController Workers
 
         public void Start()
         {
+            if (IsStarted)
+            {
+                logger.InfoFormat("Subscription already started: {0}", this);
```

Comment:
That might be better.
I don't know the project well enough yet, but is there any reason for `Subscription` to have a public constructor and `Start()` to also be public? Seems like the preferred way to create a subscription is through the factory provided by `IBus` and it always calls `Start()`.